oidc-api/src/main/java/uk/gov/di/authentication/oidc/validators/QueryParamsAuthorizeValidator.java

```diff
@@ -47,7 +47,9 @@ public Optional<AuthRequestError> validate(AuthenticationRequest authRequest) {
         ClientRegistry client = getClientFromDynamo(clientId);
 
         if (!client.getRedirectUrls().contains(authRequest.getRedirectionURI().toString())) {
-            LOG.warn("Invalid Redirect URI in request {}", authRequest.getRedirectionURI());
+            logErrorInProductionOrWarnOtherwise(
+                    String.format(
+                            "Invalid Redirect URI in request %s", authRequest.getRedirectionURI()));
```

Comment:
I imagine we want the same behaviour on this as others, even though it was warn initially?

Reply:
Could use `logErrorInProductionOrWarnOtherwise()` in `generateBadRequestResponse()`?

Reply:
Hmm, we could, but we'd likely lose a bit of information on the way (or have to do a bunch of refactoring...)